x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -38,36 +25,33 @@ import { useTimeRange } from '../../../hooks/use_time_range';
 export const chartHeight = 288;
 
 export function ServiceOverview() {
-  const { agentName, serviceName, fallbackToTransactions, runtimeName } =
-    useApmServiceContext();
+  const { agentName, serviceName } = useApmServiceContext();
 
   const {
-    query,
-    query: { environment, kuery, rangeFrom, rangeTo },
+    query: { environment, rangeFrom, rangeTo },
   } = useApmParams('/services/{serviceName}/overview');
 
   const { start, end } = useTimeRange({ rangeFrom, rangeTo });
 
-  const latencyChartHeight = 200;
-
   // The default EuiFlexGroup breaks at 768, but we want to break at 1200, so we
   // observe the window width and set the flex directions of rows accordingly
   const { isLarge } = useBreakpoints();
   const isSingleColumn = isLarge;
+
+  const latencyChartHeight = 200;
   const nonLatencyChartHeight = isSingleColumn
     ? latencyChartHeight
     : chartHeight;
-  const rowDirection = isSingleColumn ? 'column' : 'row';
-  const isRumAgent = isRumAgentName(agentName);
+  const rowDirection: RowDirection = isSingleColumn ? 'column' : 'row';
```

Comment:
I think you can use `EuiFlexGroupProps` here instead of creating your own type

```suggestion
  const rowDirection: EuiFlexGroupProps['direction'] = isSingleColumn ? 'column' : 'row';
```

Reply:
Thanks, updated in https://github.com/elastic/kibana/pull/143697/commits/8023dd8e83413a09166dfba40692a38f25d72730